src/CardSchemeComponent.res

```diff
@@ -0,0 +1,37 @@
+module CoBadgeCardSchemeDropDown = {
+  @react.component
+  let make = (~cardNumber, ~setCardBrand) => {
+    <select
+      className="w-[15px]"
+      onChange={ev => {
+        let target = ev->ReactEvent.Form.target
+        let value = target["value"]
+        setCardBrand(_ => value)
+      }}>
+      <option disabled=true> {"Select a card brand"->React.string} </option>
```

Comment:
why this option? It should be a label

Reply:
We want to show in this inside dropdown list and not as a hint in select